rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -72,14 +78,23 @@ exit 0
 %{gem_instdir}/locale
 %exclude %{gem_cache}
 %{gem_spec}
-%{foreman_bundlerd_dir}/%{gem_name}.rb
+%{foreman_bundlerd_plugin}
+%{foreman_apipie_cache_foreman}
```

Comment:
I think that macro might be wrong. If you see the `ln -s` part here: https://gist.github.com/dLobatog/f9f23eabda26b9356d41b7433a62fa5b

The second argument to `ln` is *NOT* the output directory of `plugin:apipie:cache`:
````
/builddir/build/BUILDROOT/tfm-rubygem-foreman_ansible-1.2-1.fm1_14.git.366.e7c240e.el7.centos.x86_64/opt/theforeman/tfm/root/usr/share/gems/gems/foreman_ansible-1.2/public/apipie-cache/plugin/foreman_ansible
```
but instead it's 
```
/builddir/build/BUILDROOT/tfm-rubygem-foreman_ansible-1.2-1.fm1_14.git.366.e7c240e.el7.centos.x86_64/usr/share/foreman/public/apipie-cache/plugin/foreman_ansible
```

Should I change the macro? I don't think it's linking to the right place (or outputting apipie:cache to the right place)